desc/objectives/normalization.py

```diff
@@ -38,23 +38,17 @@ def get_lowest_mode(basis, coeffs):
 
         scales["R0"] = R00
         scales["a"] = np.sqrt(np.abs(R10 * Z10))
+        scales["Psi"] = abs(thing.Psi)
+        scales["n"] = 1e19
```

Comment:
Instead of hard-coding this, compute `ne` on axis (and only do this if the equilibrium has a density profile assigned). 

Same for temperature profile.